package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "test": "jest --verbose --coverage"
```

Comment:
Tests are not working in Travis CI because of a missing "lint" script in your `package.json`